src/io/flutter/inspector/InspectorService.java

```diff
@@ -32,18 +33,60 @@
    * Group name to to manage keeping alive nodes in the tree referenced by the inspector.
    */
   private final String groupName;
-  private final FlutterDebugProcess debugProcess;
-  private final VmService vmService;
-  private final Set<InspectorServiceClient> clients;
+  @NotNull private final FlutterDebugProcess debugProcess;
+  @NotNull private final VmService vmService;
+  @NotNull private final Set<InspectorServiceClient> clients;
   private EvalOnDartLibrary inspectorLibrary;
-  private CompletableFuture<Set<String>> supportedServiceMethods;
+  @NotNull private final Set<String> supportedServiceMethods;
+
+  // TODO(jacobr): remove this field as soon as
+  // `ext.flutter.debugCallWidgetInspectorService` has been in two revs of the
```

Comment:
This is a flutter framework service extension?

Reply:
discussed offline. Switched to
ext.flutter.inspector.methodName
and cleaned up how we were passing parameters.
Parameters are now passed as either
{'arg': arg, 'objectGroup': objectGroup}
{'objectGroup': objectGroup}
or
{'arg0': arg0, 'arg1': arg1, 'arg2': arg2, ...}
for the one vararg like case.